pages/1.11/security/ent/secrets/secrets-api/index.md

```diff
@@ -45,7 +48,7 @@ All Secrets API endpoints require an authentication token.
 
 ### Via the IAM API
 
-To get an authentication token, pass the user name and password of a `superuser` in the body of a request to the `/auth/login` endpoint of the [Identity and Access Management Service API](/1.11/security/ent/iam-api/). It returns an authentication token as shown below.
+To get an authentication token, pass the user name and password of a `superuser` in the body of an request to the `/auth/login` endpoint of the [Identity and Access Management Service API](/1.11/security/ent/iam-api/). It returns an authentication token as shown below.
```

Comment:
This change from "a request" to "an request" is not right. Either "a request" or "an HTTP request". This is repeated in this document and I will not comment on it elsewhere.